docs/antora.yml

```diff
@@ -1,5 +1,5 @@
 name: rubocop-rspec
 title: RuboCop RSpec
-version: master
+version: '2.3'
```

Comment:
I trust that you know whether it should include the '.0' at the end or not ;)

Reply:
This is how we usually do it. I don't know more than that 😅

Reply:
Yeah, Only major and minor versions should be specified. Patch version is not required.
<img width="1318" alt="doc-version" src="https://user-images.githubusercontent.com/13203/116382964-637ebe00-a851-11eb-9511-1b67e2078545.png">